commands/mine_util.go

```diff
@@ -0,0 +1,135 @@
+package commands
```

Comment:
If a strong argument can be made for moving these to the Porcelain API I would be open to it. However they only appear to be used by the CLI so here they remain.

Reply:
I thought this was the whole purpose of porcelain: "// API is the porcelain implementation, a set of convenience calls written on the plumbing api, to be used to build user facing features and protocols."

All of these calls are on miner actors, so why not have it be part of miner porcelain?  We already have MinerGetOwnerAddress there.  Unless I'm missing something.

Reply:
Absolutely move them, there should be no real code in the commands. Recall that we intend to discard this CLI eventually, but still need this functionality. I think it is an error that there are any `messageQuery`'s directly in commands. cc @acruikshank